docs/components/kerbside.md

```diff
@@ -0,0 +1,83 @@
+# Kerbside
+
+Kerbside is a [SPICE](https://spice-space.org/) protocol native VDI proxy
+responsible for providing rich VDI experiences to users of Shaken Fist, oVirt,
+OpenStack, or any other cloud where the hypervisor is capable of providing SPICE
+consoles over the network.
+
+The SPICE protocol has existed for a long time ago, and still represents the
+richest and most performant option for remote desktops using Open Source
+technologies. However before Kerbside, SPICE adoption was limited because the
+console was generally provided in the form of a HTML5 transcoded interface in
+a web client. Unfortunately these HTML5 interfaces do not support many of the
+more novel features of the SPICE protocol, nor do they support high resolution
+desktops. By avoiding transcoding to a HTML5 client, we avoid these problems.
+
+Novel features of SPICE include high resolution desktops, multi-screen desktops,
+USB device passthrough, sound, multiple user connections to a single console,
+adaptive compression, and more.
+
+## Kerbside is experimental
+
+Kerbside is currently considered experimental. While it works, it has not yet
+seen large scale deployment and it is likely that it will needed modifications
+as it is hardened for production use.
+
+## Kerbside is a proxy, not a complete user interface
+
+Whilst Kerbside presents a simple administrative interface over HTTP and has
+REST APIs for orchestrating consoles, it is not intended as a complete SPICE
+desktop VDI solution. It is intended that Kerbside itself is orchestrated by
+an external system. That is, in order for a desktop to be presented to a user
+the following steps need to occur:
+
+* The user requests a desktop via an external user interface that we call "the
```

Comment:
Since these are a series of steps that must occur in order, one could consider an ordered list.